File: tnpserver/src/main/java/com/tnpserver/pojo/Partner.java
package com.tnpserver.pojo;

import jakarta.persistence.*;
import lombok.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
@Builder
public class Partner {

    private Long userId;
    private User user;
    private String organizationName;
    private String industry;
    private String organizationDescription;
    private String organizationLogoUrl;
    private String reviewsAndRatings;
}

File: tnpui/src/app/model/partner.ts
import { User } from "./user-model";

export class Partner {
  userId!: number;
  user!: User;
  organizationName!: string;
  industry!: string;
  organizationDescription!: string;
  organizationLogoUrl!: string;
  reviewsAndRatings!: string;
}
